core/SConscript.bootloader_emu

```diff
@@ -128,7 +128,6 @@ SOURCE_TREZORHAL = [
     'embed/trezorhal/unix/rng.c',
     'embed/trezorhal/unix/usb.c',
     'embed/trezorhal/unix/random_delays.c',
-    'embed/trezorhal/unix/secret.c',
```

Comment:
isn't this a problem for model R ?

Reply:
right, it's not added by the board configuration function since this is emulator ...

Reply:
yeah we need to add this conditionally in sconscript. i was thinking about something like "emulator boards", so far seems a bit like overkill but when adding new models these sconscripts might get messy, so we might eventually need that

Reply:
194f517b49df31d3f2176a1cfcb16de02fa356f8